db/db.go

```diff
@@ -2267,41 +2269,46 @@ func GetWithdrawals(query string, length, start uint64, orderBy, orderDir string
 		orderBy = "block_slot"
 	}
 
-	if query != "" {
-		bquery, _ := hex.DecodeString(strings.TrimPrefix(query, "0x"))
+	withdrawalsQuery := `
+		SELECT 
+			w.block_slot as slot,
+			w.withdrawalindex as index,
+			w.validatorindex,
+			w.address,
+			w.amount
+		FROM blocks_withdrawals w
+		INNER JOIN blocks b ON w.block_root = b.blockroot AND b.status = '1'
+		%s
+		ORDER BY %s %s
+		LIMIT $1
+		OFFSET $2`
+
+	trimmedQuery := strings.ToLower(strings.TrimPrefix(query, "0x"))
+	if trimmedQuery != "" {
+		searchQuery := ""
```

Comment:
**Minor:** There is no need for this line, just use `:=` below.

This also applies to `func GetBLSChanges`: https://github.com/gobitfly/eth2-beaconchain-explorer/blob/9e87cca798e6d7d590ab4b459d9f5acbe1db6842/db/db.go#L2874

Reply:
Right, became obsolete through changes.